package.json

```diff
@@ -18,10 +18,12 @@
     "moment": "2.24.0",
     "papaparse": "4.6.3",
     "prismjs": "1.16.0",
+    "ts-loader": "^6.2.0",
+    "typescript": "^3.6.3",
     "shelljs": "^0.8.3",
-    "slate": "0.47.4",
-    "slate-plain-serializer": "0.7.6",
-    "slate-react": "0.22.4"
+    "slate": "0.47.8",
+    "slate-plain-serializer": "0.7.10",
+    "@grafana/slate-react": "^0.22.9-grafana"
```

Comment:
super minor... but by convention, I think we should put these imports under `devDependencies` so it is more clear they are loaded as external

Reply:
I've moved them over.